lib/features/recommendation/interests/screens/interests_selection_screen.dart

```diff
@@ -99,9 +99,13 @@ class InterestsSelectionScreen extends StatelessWidget {
           floatingActionButton: Visibility(
             visible:
                 state.selectedInterests.length == InterestsState.maxInterests,
-            child: GivtFloatingActionButton(
+            child: GivtElevatedButton(
```

Comment:
also about naming - If we are going to have several buttons like primary, secondary, etc, shall we call the button like GivtPrimaryElevatedButton instead of just GivtElevatedButton?

Reply:
Primary and secondary only differ in coloring, so i dont think its enough to make 2 widgets. There is a flag to mark it as secondary

Reply:
There was a bug that would take me from orgDetailsBottomsheet to coinFlow, but I cannot recreate it, so I removed this logic. It should be in a separate PR anw